projects/developer_tooling/3Web-DAO.md

```diff
@@ -0,0 +1,75 @@
+
+## Category 
+
+infrastructure
```

Comment:
This looks like an application for me other than developer toolings. Could you please update it category and move this markdown file to `application` folder?

Reply:
move this markdown file to application folder done